src/inausoft.netCLI/Commands/HelpCommandHandler.cs

```diff
@@ -38,7 +38,7 @@ public override int Run(HelpCommand command)
 
                 foreach (var commandInfo in _configuration.CommandInfos)
                 {
-                    message.AppendLine(string.Format("{0, -15} {1}", commandInfo.Command.Name, commandInfo.Command.HelpDescription));
+                    message.AppendLine(string.Format(" {0, -15} {1}", commandInfo.Command.Name, commandInfo.Command.HelpDescription));
```

Comment:
Not sure about `{0, 15}` syntax - can you explain/provide doc link so I can understand it?

Reply:
See https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=net-5.0 - control spacing section.

Reply:
Nice one, I didn't know that! Thanks.